apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx

```diff
@@ -77,3 +77,18 @@ export const Auto = () => {
     </div>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29426&commitId=beed19b9bdd5e4dd57e171effa581812a556374f&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Square.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>